Gemfile

```diff
@@ -1,5 +1 @@
-source "https://rubygems.org"
-
-gem "excon"
-gem "pg"
-gem "sequel"
+gemspec
```

Comment:
Ah! Please keep the `source` line here - it should default to the right thing but explicit is good (: